src/components/pacbio/PacbioRunInfoEdit.vue

```diff
@@ -81,25 +83,30 @@ export default {
       type: Boolean,
     },
   },
-  data() {
-    return {
-      systemNameOptions: Object.values(PacbioRunSystems).map((system) => system.name),
-    }
-  },
   computed: {
     ...mapGetters(['runItem', 'smrtLinkVersionList', 'smrtLinkVersion']),
     smrtLinkVersionSelectOptions() {
       // Returns an array of objects with value and text properties to make
       // the options of smrt-link-version select drop-down list.
-
       return Object.values(this.smrtLinkVersionList).map(({ id, name }) => ({
         value: id,
         text: name,
       }))
     },
+    instrumentTypeSelectOptions() {
+      // Returns an array of objects with value and text properties to make
+      // the options of instrument-type select drop-down list.
+      return Object.entries(PacbioInstrumentTypes).map(([key, instrumentType]) => ({
+        value: key,
+        text: instrumentType.name,
+      }))
+    },
+    instrumentTypeValue() {
```

Comment:
TODO: This causes an issue when it is a new run it does not select the options based on the system_name